libtenzir/builtins/operators/sort.cpp

```diff
@@ -27,6 +29,46 @@ namespace tenzir::plugins::sort {
 
 namespace {
 
+auto sort_list(const series& input) -> series {
+  auto builder = series_builder{input.type};
+  for (const auto& value : values(caf::get<list_type>(input.type),
+                                  caf::get<arrow::ListArray>(*input.array))) {
+    if (not value) {
+      builder.null();
+      continue;
+    }
+    auto materialized = materialize(*value);
+    std::ranges::sort(materialized);
+    builder.data(materialized);
```

Comment:
@IyeOnline I wasn't sure whether there perhaps is a way to avoid materializing here with the series builder. (It's not critical, I am mostly curious.)

Reply:
We discussed this, turns out we cannot do this easily.